emission/core/wrapper/suggestion_sys.py

```diff
@@ -437,23 +438,32 @@ def review_start_loc_nominatim(lat, lon):
 11: "AIR_ON_HSR"
 '''
 
-def calculate_yelp_server_suggestion_singletrip_nominatim(tripid):
-    return_obj = { 'message': "Good job walking and biking! No suggestion to show.", 'method' : 'bike'}
-    start_lat, start_lon = geojson_to_lat_lon_separated(tripid["start_loc"])
-    end_lat, end_lon = geojson_to_lat_lon_separated(tripid["end_loc"])
+
+def calculate_yelp_server_suggestion_singletrip_nominatim(tripid, uuid):
```

Comment:
nit: IIRC, wrt coding convention, I typically put the uuid as the first argument, since it is implied that it is for that user.